src/unitxt/operators.py

```diff
@@ -100,6 +100,72 @@ def process(self, instance: Dict[str, Any], stream_name: str = None) -> Dict[str
         return instance
 
 
+class MapInstanceMultiLabelValues(StreamInstanceOperator):
+    """A class used to map instance values into a stream.
+
+    This class is a type of StreamInstanceOperator,
+    it maps values of instances in a stream using predefined mappers.
+
+    Attributes:
+        mappers (Dict[str, Dict[str, str]]): The mappers to use for mapping instance values.
+            Keys are the names of the fields to be mapped, and values are dictionaries
+            that define the mapping from old values to new values.
+        strict (bool): If True, the mapping is applied strictly. That means if a value
+            does not exist in the mapper, it will raise a KeyError. If False, values
+            that are not present in the mapper are kept as they are.
+        process_every_value (bool): Let 'm' denote the mapper associated with field 'f', and let 'v' be a key in 'm'.
```

Comment:
Better to avoid formal notations and simplify for the user.  .  All this means, is that if prices_every_value =True is that, each field mist be a  list, and the mapping is done to individual elements in the list.